charts/kyverno/templates/admission-controller/deployment.yaml

```diff
@@ -82,6 +82,7 @@ spec:
           imagePullPolicy: {{ default .Values.admissionController.container.image.pullPolicy .Values.admissionController.initContainer.image.pullPolicy }}
           args:
             - --loggingFormat={{ .Values.admissionController.logging.format }}
+            - --v={{ .Values.admissionController.logging.verbosity }}
```

Comment:
Is it a single dash to set the log level?

Reply:
Nope, golang supports both single and double dash.